torch/_inductor/codegen/cpp_wrapper_gpu.py

```diff
@@ -549,27 +549,28 @@ def generate_kernel_call(
                 device_index, call_args
             )
             kernel_var_name = self.generate_load_kernel_once(kernel_name, V.graph)
-
-            # args with value 1 are added into equal_to_1 and constants
-            # in triton_meta (in the Python codegen) which makes them
-            # inlined in the PTX and compiled CUBIN
-            arg_signatures = []
-            if (
-                triton_meta is not None
-                and triton_meta.get("configs")
-                and triton_meta.get("signature")
-            ):
-                equal_to_1 = triton_meta["configs"][0].equal_to_1
-                call_args = [
-                    arg for i, arg in enumerate(call_args) if i not in equal_to_1
-                ]
-                arg_types = [t for i, t in enumerate(arg_types) if i not in equal_to_1]
-                # extract the arg signatures from triton_meta
-                arg_signatures = triton_meta["signature"].values()
-                arg_signatures = [
-                    v for i, v in enumerate(arg_signatures) if i not in equal_to_1
-                ]
-
+            if triton_version_uses_attrs_dict():
+                arg_signatures = [val for key, val in triton_meta["signature"].items() if val != 'constexpr']
```

Comment:
The new version does not have `equal_to_1` field, and the signature also contains `constexpr` expressions that the wrapper does not count on.